packages/connect/src/base/listener.ts

```diff
@@ -491,7 +424,7 @@ class Listener extends EventEmitter implements InterfaceListener {
     log('new inbound connection %s', maConn.remoteAddr)
 
     try {
-      conn = await this.upgrader.upgradeInbound(maConn)
+      await this.upgradeInbound(maConn)
```

Comment:
changed libp2p internals. `upgradeInbound` will automatically emit internal libp2p events, so no need to return the connection afterwards